yaserde/src/lib.rs

```diff
@@ -13,7 +13,7 @@
 //! file for microcontrollers is called [SVD](https://github.com/rust-embedded/svd/)
 //! and it can be defined on YaSerDe via structs like so:
 //!
-//!```
+//!```verbatim
```

Comment:
use rust instead

Reply:
I tried to use the `rust` markdown tag at first, but then rustc actually tries to parse the fragments, failing miserably while building the docs: https://travis-ci.com/github/media-io/yaserde/jobs/481193204#L450

Couldn't easily find any rustdoc macro to avoid that behaviour :/

Reply:
Nevermind, fixed that now via https://doc.rust-lang.org/rustdoc/documentation-tests.html#attributes `ignore`, thanks to @vickenty 👍🏻

Reply:
I have updated the example, it missed the struct `DevAttrs` ...